rebar.config

```diff
@@ -1,5 +1,6 @@
 {deps, [{kafka_protocol, {git, "https://github.com/emqx/kafka_protocol", {tag, "2.3.6.1"}}},
-        {replayq, {git, "https://github.com/emqx/replayq", {tag, "0.3.1"}}}
+        {replayq, {git, "https://github.com/emqx/replayq", {tag, "0.3.1"}}},
```

Comment:
just a reminder: 
```suggestion
        {replayq, {git, "https://github.com/emqx/replayq", {tag, "0.3.2"}}},
```

Reply:
ok, if we do want to bump replayq version, I could bring in this change as well:
https://github.com/emqx/replayq/pull/10